README.md

```diff
@@ -126,20 +126,23 @@ The commands will do the following:
 </details>
 <br>
 
-Connect VS Code to your JFrog Platform instance by clicking on the green Connect ![Connect](resources/readme/connect.png) button or the provided button in the JFrog extension tab:
-![Connect](resources/readme/preview/connect.png)
+Once you installed, You can view the sigh in page by clicking on the Frog extension tab:
+![LoginPage](resources/readme/preview/sighInPage.png)
 
-You can choose one of the following methods to provide VS Code with the JFrog Platform connections details.
+
+Enter your JFrog platform connection details. Switching from Access token to Username and password can be done by clicking the `Using Password` button.
+
+**Note**: If you would like to use custom URLs for Artifactory or Xray, click on 'Advanced'
+
+There are other methods of providing your JFrog Platform credentials to VS Code than manually writing them in:
```

Comment:
There are other methods of providing your JFrog Platform credentials to VS Code than manually writing them in:
-->
You can also authenticate with your JFrog Platform instance by using the configured [JFrog CLI's Connection Details] or by [Using Environment Variables].

(Replace the above sections enclosed with square brackets with links to the below corresponding documentation sections)